testsuite/features/support/lavanda.rb

```diff
@@ -65,4 +65,14 @@ def run_until_fail(cmd)
       result
     end
   end
+
+  def wait_while_process_running(process)
+    result = nil
+    repeat_until_timeout(report_result: true) do
+      result, code = run("pgrep -x #{process} >/dev/null", false)
+      break if code.zero?
```

Comment:
If the command returns nonzero status, then there is no such a process running (thus we are all set):
`break if code.zero?` -> `break unless code.zero?`

Reply:
Right, my bad, thank you!